MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
```

Comment:
I haven't seen many cases which have to run multiple concurrent builds for a same job, yet.
So preferred to disable this in the template to avoid potential risk for any single job to burn out test resources by misconfiguring its trigger.